### PR TITLE
chore(js): Upgrade compression-webpack-plugin to 1.1.11

### PR DESCRIPTION
**Pull request by billyvg**
Has some minor performance benefits (small sample size but maybe a 2-3% speed boost), and better webpack4 support

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "bootstrap": "3.3.5",
     "classnames": "2.2.0",
     "clipboard": "^1.7.1",
-    "compression-webpack-plugin": "^1.0.0",
+    "compression-webpack-plugin": "1.0.0",
```
Comment (billyvg) [inline package.json]: er what

**Comment by markstory** [comment]
Is this something we're still interested in pursuing or can it be closed because it can no longer be merged?

**Comment by billyvg** [comment]
Yeah let's land this

**Review by markstory — APPROVED** [review]
Looks like a higher version :+1:

**Pull request merged**